packages/server-core/src/social/invite/invite.class.ts

```diff
@@ -199,6 +199,8 @@ export const inviteAll = async (inviteService: Invite, query: Query, user: UserT
   }
   if (!query.existenceCheck) delete query.userId
   delete query.existenceCheck
+  delete query.search
```

Comment:
Maybe its good to have a check here like

```
if (query.search) {
  delete query.search
}
```

Reply:
yes

Reply:
Actually it was the reason for the bug
<img width="479" alt="image" src="https://github.com/EtherealEngine/etherealengine/assets/55396651/469c72a2-f9f2-4638-9cdd-163b719a595a">

Reply:
Yes, it can be removed. But just put that delete in an if condition. I think as you merged the PR with latest dev that change is no longer required?

Reply:
Yes, you are right
That code is now present in latest dev
Maybe we could add the tests in this PR only

Reply:
Yes, you are right
That code is now present in latest dev
Maybe we could add the tests in this PR only